### ISSUE TITLE
COLOR: Bugs

### ISSUE THREAD

**Issue by Brahmanathaswami** [Himalayan-Academy/Siva-Siva-App#181]
- [x]  hangs on first screen unless the user touches it, this is a confusing behavior as there is no indication that the user needs to touch anything.
- [x] information modal can only be closed by pressing the "I" icon again, an X should be provided near the top of the model which is what the user expects.
- [x] in the configuration, the "Time Commitment:" label is smaller than the text, which leads to clipping (see shot)
- [ ] takes multiple attempts to try to stop color meditation.

**Comment by Brahmanathaswami** [comment]
solved the timing problem, we can close this for now.

**Comment by Brahmanathaswami** [comment]
Heidi says:

Aum Swami,

The pause in primary color meditation didn't appear to pause, but as I kept trying as the program kept playing it would jump back to starting red screen, pause, then continue again.

Went into tertiary colors, started hitting pause, felt like phone was in ever more confusion and then stopped.

I tried changing settings but couldn't.  Feels like pause has possibly covered.

Journal entries looked good on first several screens.  When I scrolled further back saw what looked like code overwriting entries. ( Last time it was entries overwriting entries)

Like different colors on favorites tab in the different screens.

Was confused at first on how to exit favorites. Different words indicating main journal would help me.

Aum Shanti,

Heidi
